diff -Nru 0verkill-0.16/cfg.h 0verkill-0.16.new/cfg.h
--- 0verkill-0.16/cfg.h	Sat Mar 15 13:01:51 2003
+++ 0verkill-0.16.new/cfg.h	Sat Mar 15 13:08:52 2003
@@ -32,6 +32,7 @@
 #define LEVEL_SPRITES_SUFFIX ".sp"
 
 #define CFG_FILE ".0verkill"
+#define ETC_DIR_CFG_FILE "0verkill"
 
 #define AREA_X 1000
 #define AREA_Y 500
diff -Nru 0verkill-0.16/client.c 0verkill-0.16.new/client.c
--- 0verkill-0.16/client.c	Sun Dec 16 10:26:38 2001
+++ 0verkill-0.16.new/client.c	Sat Mar 15 13:12:35 2003
@@ -189,8 +189,12 @@
 	FILE *stream;
 	int a;
 	unsigned char txt[256];
-
+	char *etc_dir;
 #ifndef WIN32
+	etc_dir = getenv("ETC_DIR");
+	if (etc_dir) {
+		sprintf(txt, "%s/%s", etc_dir, ETC_DIR_CFG_FILE);
+	} else
 	sprintf(txt,"%s/%s",getenv("HOME"),CFG_FILE);
 #else
 	sprintf(txt,"./%s",CFG_FILE);
@@ -221,8 +225,13 @@
 {
 	FILE *stream;
 	unsigned char txt[256];
-
+	char *etc_dir;
+	
 #ifndef WIN32
+	etc_dir = getenv("ETC_DIR");
+	if (etc_dir) {
+		sprintf(txt, "%s/%s", etc_dir, ETC_DIR_CFG_FILE);
+	} else
 	sprintf(txt,"%s/%s",getenv("HOME"),CFG_FILE);
 #else
 	sprintf(txt,"./%s",CFG_FILE);
